doc/nrf/known_issues.rst

```diff
@@ -2750,17 +2762,17 @@ SoftDevice Controller
 The issues in this section are related to :ref:`nrfxlib:softdevice_controller`.
 In addition to the known issues listed here, see also :ref:`softdevice_controller_limitations` for permanent limitations.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0 v2-3-0
 
 DRGN-18424: Rare assert when stopping the Periodic Advertiser role
   In rare cases, when a Periodic Advertiser instance is being stopped while another Periodic Advertiser instance is still running, the controller can assert.
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
 
 DRGN-18424: Rare assert when stopping the Periodic Sync role
   In rare cases, when a Periodic Sync instance is being stopped while another Periodic Sync instance is still running, the controller can assert.
 
-.. rst-class:: v2-3-0
+.. rst-class:: v2-4-0 v2-3-0
 
```

Comment:
Some other issues are missing here:

rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-18714: Assert when advertising or initiating after disconnection 
The controller could assert when starting a connectable advertiser or creating a connection too quickly after disconnection.

rst-class:: v2-3-0 v2-2-0
DRGN-18775:  Periodic Advertisement Sync Transfer receiver fails to synchronize with large periodic advertising intervals
The Periodic Advertisement Sync Transfer (PAST) sender may generate an incorrect SyncInfo field for periodic advertising intervals greater than 5 seconds.

rst-class:: v2-3-0 v2-2-0
DRGN-18833:  Assert when the periodic sync is in progress
The Periodic Advertisement Sync Transfer (PAST) sender could assert if the associated periodic sync was not fully established.

rst-class:: v2-3-0 v2-2-0
DRGN-18971: Low TX power on the nRF21540 DK in connected state
TX power value goes to the default one on the nRF21540 DK in connected state when using MPSL FEM and manually configuring the radio power.

rst-class:: v2-3-0 v2-2-0
DRGN-19003: Fail to synchronize using the Periodic Advertising Sync Transfer procedure
The controller cannot synchronize to a periodic advertising train using the Periodic Advertising Sync Transfer procedure if it has previously tried to do it while it was already synchronized to the periodic advertising train.

rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-18840: Dicsonnect with transaction_collision when receiving LL_PHY_UPDATE_IND with "no change"
The peripheral would disconnect with DIFFERENT_TRANSACTION_COLLISION when a collision of a connection update and a PHY update occurs even if the central asks for no change.

rst-class:: v2-3-0 v2-2-0 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0
DRGN-19058: Do not accept the advertising handle specified by the host
The Controller would not accept an advertising handle provided in HCI commands with values above the configured number of advertising sets.